CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ Upgrade nginz (#1658)
 
 ## Federation changes (alpha feature, do not use yet)
 
+* Added documentation of federation errors (#1674).
```

Comment:
This is a federation-api-backwards-incompatible change (since router.proto changes), which is fine right now, but we'd need to be more careful about this in the future. Could be added to the changelog like here: https://github.com/wireapp/wire-server/blob/develop/CHANGELOG.md#internal-federation-api-changes

Reply:
If we had kept the existing field indices, then I think this sort of change should be backwards compatible. Is this right?

Reply:
Yes, if the meaning and the numbers stay, but only things are added, it's backwards compatible. If indices change or are removed it's not.
So far so good, no need for backwards compatibility concerns just yet.